### ISSUE TITLE
Improvement- page csv download button

### ISSUE THREAD

**Issue by pengchenglai** [openforis/fra-platform#2804]
- [ ] Next to Section title, add `(icon download) CSV - All tables ` button that creates a zip file with all section CSV Tables data 

@minotogna 
In the case of multiple tables in one session, plz add a page button which pack up all tables info.
![1](https://github.com/openforis/fra-platform/assets/99216009/1a3eb41a-cc48-4393-bb1c-ccdf835b7994)

**Comment by minotogna** [comment]
> @minotogna 
> 
> In the case of multiple tables in one session, plz add a page button which pack up all tables info.
> 
> ![1](https://github.com/openforis/fra-platform/assets/99216009/1a3eb41a-cc48-4393-bb1c-ccdf835b7994)
> 
> 
> 
> 

@pengchenglai Can you please elaborate further ?

**Comment by pengchenglai** [comment]
Like there is a CSV tab that allows users to download all the CSV files in one session.

For instance, all the tables shown below would wrap up in one .csv file.
![1](https://github.com/openforis/fra-platform/assets/99216009/c12d8f4e-03ab-492c-b350-a69e95467827)
![2](https://github.com/openforis/fra-platform/assets/99216009/1e6a98cf-9705-408e-b165-61c8d8e4754b)

**Comment by minotogna** [comment]
Ok @pengchenglai I consider this low priority

**Comment by pengchenglai** [comment]
@minotogna 

Right, without specific notice(in the title) the issue normally is not very urgent.
